.nyc_output/processinfo/index.json

```diff
@@ -0,0 +1 @@
+{"processes":{},"files":{},"externalIds":{}}
```

Comment:
Add EOL here

Reply:
Add EOL here. What is EOL? How do I add it?